internal/raft/append_entries.go

```diff
@@ -6,6 +6,52 @@ import "github.com/tomarrell/lbadd/internal/raft/message"
 // to the follower node. This function generates the response to be sent to the leader node.
 // This is the response to the contact by the leader to assert it's leadership.
 func AppendEntriesResponse(node *Node, req *message.AppendEntriesRequest) *message.AppendEntriesResponse {
```

Comment:
Does this have to be a function? Why can't it be a method?
`node` seems like a receiver, and `req` like the parameter.

If this has to be a function, rename it to `NewAppendEntriesResponse`.

Reply:
It responds to Append Entries.

Let this merge into raft, then I can see how it can be converted to a method.
Needs some more work.

Reply:
Ok for me, since this doesn't merge into `master`